test/SILOptimizer/devirt_access.swift

```diff
@@ -4,18 +4,6 @@
 // RUN: %target-swift-frontend -O -primary-file %s %S/Inputs/devirt_access_helper.swift -I %t -emit-sil -sil-inline-threshold 1000 -sil-verify-all | %FileCheck -check-prefix=WHOLE-MODULE %s
 // RUN: %target-swift-frontend -O -primary-file %s %S/Inputs/devirt_access_helper.swift -I %t -emit-sil -sil-inline-threshold 1000 -sil-verify-all | %FileCheck -check-prefix=PRIMARY-FILE %s
 
-import devirt_access_other_module
```

Comment:
I don't think it's a good idea to just delete part of the test. Maybe run it under explicit non-resilient mode?